core/arch/arm/plat-rpi3/platform_flags.mk

```diff
@@ -0,0 +1,20 @@
+# 32-bit flags
```

Comment:
I think it's not used since 739804b and `ta/ta.mk` should be updated accordingly (i.e., delete the line you're mentioning). @jenswi-linaro is this correct?